spec/lib/areacodes_spec.rb

```diff
@@ -14,4 +14,93 @@
       expect(Geolookup::USA::AreaCodes.find(2023)).to eql({})
     end
   end
+
+  describe '.details' do
+    it 'is not empty' do
+      expect(described_class.details).to_not be_empty
+    end
+
+    it 'is over 9000' do # https://www.youtube.com/watch?v=SiMHTK15Pik
+      expect(described_class.details.size>9000).to be_truthy
+    end
+
+    it 'uses same list' do
+      expect(described_class.details.object_id)
+        .to eql(described_class.details.object_id)
+    end
+
+    it 'has valid records' do
+      described_class.details.each do |r|
+        expect(r.areacode>200).to be_truthy, "Failed areacode: #{r}"
+        expect(r.city).to_not be_empty, "Failed city: #{r}"
+        expect(r.statecode>0).to be_truthy, "Failed state: #{r}"
+        expect(r.countycode>0).to be_truthy, "Failed county: #{r}"
+        expect(r.population>0).to be_truthy, "Failed population: #{r}"
+      end
+    end
```

Comment:
Nice!  That's a comprehensive test and should catch ETL problems if/when the file gets updated.

Reply:
The file will never get updated.

Reply:
kidding.